CMakeLists.txt

```diff
@@ -155,7 +155,7 @@ IF(BUILD_PYTHON_INTERFACE)
   # Include and add the bindings directory
   INCLUDE_DIRECTORIES(bindings)
   ADD_SUBDIRECTORY(bindings)
-  ADD_SUBDIRECTORY(unittest/python/${PROJECT_NAME})
+  ADD_SUBDIRECTORY(unittest/python/${PROJECT_NAME}_legacy)
```

Comment:
This needs to be reverted since it doesn't include the Python unittest, which doesn't have anything to do with legacy unittest. Furthermore, there is no `crocodyl_legacy` folder and we don't want to build/unittest legacy code.

There is another revert to be done regarding this topic, see the comment below.

Reply:
done